scripts/operation/docker-compose.yml

```diff
@@ -139,15 +136,17 @@ services:
       - "${STORAGE_SERVICE_PORT}:${STORAGE_SERVICE_PORT}"
 
   frontend:
-    image: trubudget/frontend:${TAG}
+    # image: trubudget/frontend:${TAG}
+    # ToDo: Replace local build with image from dockerhub. Local image must be used because of new/renamed env vars (API_HOST, API_PORT)
+    # image: trubudget/frontend:${TAG}
+    build:
+      context: ../../frontend
     environment:
-      NODE_ENV: ${NODE_ENV}
+      NODE_ENV: production
       REACT_APP_EXPORT_SERVICE_ENABLED: ${REACT_APP_EXPORT_SERVICE_ENABLED}
       REACT_APP_EMAIL_SERVICE_ENABLED: ${REACT_APP_EMAIL_SERVICE_ENABLED}
-      PROD_API_HOST: master-api
-      PROD_API_PORT: ${API_PORT}
-      TEST_API_HOST: master-api
-      TEST_API_PORT: ${API_PORT}
+      API_HOST: master-api
+      API_PORT: ${API_PORT}
```

Comment:
This todo should be resolved after this PR is merged, because the env var names are hardcoded in the images of dockerhub.